### PR TITLE
Add e2e tests for pruning resources

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan [myan@redhat.com](mailto:myan@redhat.com)
Fixed: https://github.com/stolostron/backlog/issues/25741

### CODE REVIEW

File: cicd-scripts/run-local-e2e-test.sh [hunk 1]
```diff
@@ -93,7 +93,7 @@ done
 verbose=${verbose:=5}
 
 if [ -z "${filter}" ]; then
-  ginkgo --output-dir="${ROOT_DIR}/test/resources/report" --json-report=report.json \
+  ginkgo --label-filter="!e2e-tests-prune" --output-dir="${ROOT_DIR}/test/resources/report" --json-report=report.json \
```
Comment (morvencao) [inline cicd-scripts/run-local-e2e-test.sh:96]: it will not run in normal case?

Comment (yanmxa) [reply cicd-scripts/run-local-e2e-test.sh:96]: Yes. 
The`e2e-tests-prune` will delete the manager, Which will block other e2e tests.

**Review by morvencao — COMMENTED** [review]
/lgtm

**Pull request merged**